test/ServerSpec.js

```diff
@@ -96,50 +109,76 @@ describe('Unprotected routes: ', function() {
       });
     });
 
-    it('should return empty response for nonexistent users', function(done) {
-      request(loginOptions, function(err, res, body) {
+  });
+
+  describe('Memory creation', function() {
+   
+    // note: need aws credentials set up (in the current tab) for this to work
+    it ('should create a local file and push it to s3', function(done) {
```

Comment:
Should this test be named 'should return 201 when uploading image? This currently doesn't check for local files or S3